plugins/vscode/test/helper/index.ts

```diff
@@ -0,0 +1,34 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+
+export let doc: vscode.TextDocument;
+export let editor: vscode.TextEditor;
+export let documentEol: string;
+export let platformEol: string;
+
+/**
+ * Activates the Predicate extension
+ */
+export async function activate(docUri: vscode.Uri) {
+  // The extensionId is `publisher.name` from package.json
+  const ext = vscode.extensions.getExtension('teleport.predicate')!;
+  await ext?.activate();
+  try {
+    doc = await vscode.workspace.openTextDocument(docUri);
+    editor = await vscode.window.showTextDocument(doc);
+    await sleep(2000);
+  } catch (e) {
+    console.error(e);
+  }
+}
+
+export async function sleep(ms: number) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+export const getDocPath = (p: string) => {
+  return path.resolve(__dirname, '../../../test/testFixture', p);
+};
```

Comment:
inconsistent line spacing? Have we can something like prettier over this? Also, can we add prettier format checks to the CI when introducing this?

Reply:
Prettier is already configured in these source files. Eslint is setup according Google typescript lint config [google gts](https://github.com/google/gts). But I will check once with our frontend team on style guide. 
Definitely, I have created this https://github.com/gravitational/predicate-lang/issues/75 to setup formatter and linter for Nodejs and python. i will create PR for that separately.

Reply:
@xacrimon I have rerun prettier and linter on all the files and I have also added new workflow that tests and runs linter on each push.